rpms/EL9/archivematica-storage-service/package.spec

```diff
@@ -90,7 +99,7 @@ fi
 mkdir -p /var/log/archivematica/storage-service /var/archivematica/storage-service /var/archivematica/storage_service
 touch /var/log/archivematica/storage-service/storage_service.log
 touch /var/log/archivematica/storage-service/storage_service_debug.log
-chown -R archivematica:archivematica /var/archivematica/storage_service /var/log/archivematica/storage-service /usr/lib/archivematica/storage_service /var/lib/archivematica /var/archivematica/storage-service
+chown -R archivematica:archivematica /var/archivematica/storage_service /var/log/archivematica/storage-service /var/lib/archivematica /var/archivematica/storage-service /opt/archivematica/archivematica-storage-service
```

Comment:
Why is the application code owner changed to archivematica?

Reply:
From my pov, this introduces a security risk, as the "archivematica" user will be able to change the code installed from the rpm's.

Reply:
It's not me the one that does that consideration, but  [CWE-732](https://cwe.mitre.org/data/definitions/732.html)  . Seems that ansible is on the same boat.

Reply:
Sure, I understand your point and I've removed the source code directory in https://github.com/artefactual-labs/am-packbuild/pull/392/commits/4f99b2d79c609e75a2f7c78bd8411d7dfc251566.